blockchain/client.go

```diff
@@ -12,11 +14,39 @@ import (
 	"github.com/ethereum/go-ethereum/rpc"
 )
 
+// Receipt extends go-ethereum/core/types/Receipt struct with BlockNumber
 type Receipt struct {
 	*types.Receipt
 	BlockNumber string
 }
 
+// UnmarshalJSON implement json/Unmarshaler interface (stdlib)
+// reference: go-ethereum/core/types/Receipt has custom Unmarshaler for casting for its own types
```

Comment:
Do not specify **what** happen into a method, specify **why do we need to do this**.
I personally feel that this comment is weird and meaningless.